packages/geany.rb

```diff
@@ -3,32 +3,40 @@
 class Geany < Package
   description 'Geany is a small and lightweight Integrated Development Environment.'
   homepage 'https://www.geany.org/'
-  version '1.37.1'
+  version '1.37.1-b524'
   license 'GPL-2+ HPND'
   compatibility 'all'
-    source_url 'https://download.geany.org/geany-1.37.1.tar.bz2'
-    source_sha256 '18c5756444c1d8bcd737c8ecfd4ef0b3607c924fc02560d4e8b78f6121531a18'
-
-  binary_url ({
-     aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/geany/1.37.1_armv7l/geany-1.37.1-chromeos-armv7l.tar.xz',
-      armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/geany/1.37.1_armv7l/geany-1.37.1-chromeos-armv7l.tar.xz',
-        i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/geany/1.37.1_i686/geany-1.37.1-chromeos-i686.tar.xz',
-      x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/geany/1.37.1_x86_64/geany-1.37.1-chromeos-x86_64.tar.xz',
+  source_url 'https://github.com/geany/geany.git'
+  git_hashtag 'b524a58e12e85c94a32f64fb72615978b2628af1'
+
+  binary_url({
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/geany/1.37.1-b524_armv7l/geany-1.37.1-b524-chromeos-armv7l.tpxz',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/geany/1.37.1-b524_armv7l/geany-1.37.1-b524-chromeos-armv7l.tpxz',
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/geany/1.37.1-b524_i686/geany-1.37.1-b524-chromeos-i686.tpxz',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/geany/1.37.1-b524_x86_64/geany-1.37.1-b524-chromeos-x86_64.tpxz'
   })
-  binary_sha256 ({
-     aarch64: '23569bccfea77b83925f8d8b69b0726956fab967588f8b87433a9aec9e846f9c',
-      armv7l: '23569bccfea77b83925f8d8b69b0726956fab967588f8b87433a9aec9e846f9c',
-        i686: '7a8600c481502fabad7079513381b21b9d3d628bcbc9c24d59ef3825e3b49459',
-      x86_64: '02fb7bbd757af647903507b68d880a5207c3666725f49748edb1e71c0c629a51',
+  binary_sha256({
+    aarch64: '4533890949b4dd456331fa10e2482ad59a93a28a43b9e413bacf22abab0f4fcc',
+     armv7l: '4533890949b4dd456331fa10e2482ad59a93a28a43b9e413bacf22abab0f4fcc',
+       i686: '8f49d7208ed72ca166b899b75031947b4e40fd8c9efcfb6cb91eb9b4df512de3',
+     x86_64: '0b1870b7fc669a731d88ea4be18b5fba2ef7e233a5ffb1483aaa3cb3d710e78b'
   })
 
-    depends_on 'gtk3'
-    depends_on 'xdg_base'
-    depends_on 'sommelier'
+  depends_on 'gtk3'
+  depends_on 'xdg_base'
+  depends_on 'sommelier'
 
+  @xdg_config_home = ENV['XDG_CONFIG_HOME']
```

Comment:
Doesn't this make an assumption the environment variable is set?  What if it isn't set?